core/dbt/config/runtime.py

```diff
@@ -353,7 +353,7 @@ def load_projects(
                 yield project.project_name, project
 
     def _get_project_directories(self) -> Iterator[Path]:
-        root = Path(self.project_root) / self.modules_path
+        root = Path.cwd() / self.modules_path
```

Comment:
A little concern with possible behavior this line might introduce. I think the assumption here is that `load_dependencies` will be called after moving to the root project dir. I _think_ it is fine, will defer to you!